tech-spec.md

```diff
@@ -0,0 +1,31 @@
+# Tech Spec
+![overview](https://i.imgur.com/hbz9N5L.png)
+
+## Components
+
+### React
+Providing the dynamic functionality for our project showcase.
+Handles all form submissions.
+* Project Showcase page showing all projects
+* Individual project detail page for each project
+* Volunteer form on each individual project
+  - Store info in local storage.
+
+### Nginx
+Serving the static Gatsby pages.
+
+### Node server
+* Express for routing/api.
+* Typescript is the language choice.
+* Send email using Gmail API.
+  - Sent when project are submitted and when volunteers sign up.
+* Slack integreation when new projects are proposed.
+
+### Postgres DB
+To store our project info.
+
+### Gatsby
+Where our current static pages are made.
+
+### Kubernetes
+Hosting everything but Gatsby.
```

Comment:
The Gatsby stuff is already hosted in our Kubernetes cluster. To clarify, Gatsby is a tool to generate static web pages from Markdown, etc. The *generated* pages are hosted in a container in our kubernetes cluster. That container holds nginx and the static assets generated by Gatsby. But the code generation part of gatsby isn't really hosted anywhere so this sentence is a little confusing.